src/test/regress/output/multi_copy.source

```diff
@@ -304,6 +304,28 @@ SELECT count(*) FROM pg_dist_shard WHERE logicalrelid = 'lineitem_copy_append'::
      5
 (1 row)
 
+-- Test round robin shard policy
```

Comment:
Looking into this, I noticed that local-node-first does not care about the port numbers and only checks the host name in WorkerGetNodeWithName() function. It seems testing this on a single machine does not make sense.